src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,23 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  (comments.length > 0)
```

Comment:
Condition statements should be wrapped with curly braces for readability purposes even though it's not mandatory in JSX. This will help other developers understand your code more easily.